apps/01PigLatin.js

```diff
@@ -7,10 +7,86 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
+  // Concept Check
+  //
+  // Please explain why we are using String.prototype.toLowerCase() here.
+  // For more info:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+  //
+  // Your Answer: To convert the string value to lowercase. This way, when we run the app, the string value is converted to lowercase regardless of what is typed in.
+  //
+
+  word = word.toLowerCase();
+
+  // Concept Check
+//
+// What is the significance of setting vowelIndex to -1?
+// This might help:
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/lastIndexOf
+//
+// Your Answer: When the app runs, it searches the string value from back to front, starting from 'fromIndex.' It returns -1 if no value is found, i.e., there are no vowels.
```

Comment:
Good job. It looks like i put the wrong link in here.

Should have put this one to [MDN String.prototype.indexOf()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf). Oops.